node/parachain/src/chain_spec.rs

```diff
@@ -53,8 +72,14 @@ pub fn get_chain_spec(para_id: ParaId) -> ChainSpec {
 		ChainType::Local,
 		move || {
 			testnet_genesis(
-				AccountId::from_str("6Be02d1d3665660d22FF9624b7BE0551ee1Ac91b").unwrap(),
-				vec![AccountId::from_str("6Be02d1d3665660d22FF9624b7BE0551ee1Ac91b").unwrap()],
+				// Sudo Account
+				get_account_id_from_seed::<ecdsa::Public>("Alice"),
```

Comment:
You are using the Substrate ECDSA here which doesn't derive the seed to a Ethereum public address but to a Substrate public address. This won't be compatible with Ethereum/Moonbeam tools.

Reply:
I'm not sure how to proceed. I can import these accounts into Apps via private key and use them. But I agree Apps cannot derive them from mnemonic. What should we be doing?

Reply:
One solution is to skip the derivation part:
 We derive Alice, Bob using substrate and store those PK/Pub in the code.
 We store those PK/Pub directly in polkadotJs if that is possible (@joelamouche).
 
I don't have a better idea so far to achieve the Alice/Bob...

Reply:
The way I see it there are two possible solutions:
- We want Alice and Bob to derive from Ethereum derivation and we take the addresses simply derived from "bottom drive obey lake curtain smoke basket hold race lonely fit walk" in a canonic way, like in ganache-cli. Alice the address 0, Bob address 1, etc. We use the private keys generated bellow to derive Alice and Bob in rust and in the pokkadot-js suite.
<img width="965" alt="Screenshot 2021-01-28 at 10 42 27" src="https://user-images.githubusercontent.com/16758383/106119564-a26b7d80-6155-11eb-90c7-01451245945c.png">

- We don't mind using non-ethereum derivation and we keep non-ethereum derivation in rust and polkadot-js to derive Alice and Bob, mapping the substrate address to an ethereum address and using the key pairs everywhere else, as suggested in @crystalin 's previous comment

Reply:
I'm leaning toward the second option, or just staying away from alice and bob altogether. I see two real cons to doing ethereum-style derivation:
1. We would have to figure out how to do it in Rust (maybe could copy from open ethereum?)
2. We would need to figure out how to do it in Polkadot js to Jaco's liking.

Reply:
>We would have to figure out how to do it in Rust 

We would be using the private keys generated elsewhere from the mnemonic

>We would need to figure out how to do it in Polkadot js to Jaco's liking

Yes that is indeed the biggest unknown... But basically either he won't let us change the dev addresses for Moonbeam or he will and those could be reused for any ethereum based parachain